packit/upstream.py

```diff
@@ -617,21 +617,28 @@ def create_rpms(self, rpm_dir: Union[str, Path] = None) -> List[Path]:
                 f"The `rpmbuild` command failed:\n{ex}"
             ) from ex
 
-        rpms = self._get_rpms_from_rpmbuild_output(out)
+        rpms = Upstream._get_rpms_from_rpmbuild_output(out)
         return [Path(rpm) for rpm in rpms]
 
-    def _get_rpms_from_rpmbuild_output(self, output: str) -> List[str]:
+    @staticmethod
+    def _get_rpms_from_rpmbuild_output(output: str) -> List[str]:
         """
         Try to find the rpm files in the `rpmbuild -bb` command output.
 
-        :param output: output of the `rpmbuild -bb` command
-        :return: the names of the RPM files
+        Args:
+            output: Output of the `rpmbuild -bb` command.
+
+        Returns:
+            List of names of the RPM files.
         """
         logger.debug(f"The `rpmbuild` command output: {output}")
-        reg = r": (.+\.rpm)"
-        logger.debug(re.findall(reg, output))
+        reg = r": (.+\.rpm)(\s+|$)"
```

Comment:
If the `(\s+|$)` part is just for anchoring, wouldn't `(\s|$)` work equally well?

For the filename wouldn't it be better to use `[^\s]+\.rpm` (not-space characters followed by `.rpm`)?

Reply:
> If the `(\s+|$)` part is just for anchoring, wouldn't `(\s|$)` work equally well?

I'm not sure if I want to rely on expecting `\n` only.

> For the filename wouldn't it be better to use `[^\s]+\.rpm` (not-space characters followed by `.rpm`)?

Should be better, thanks :)